docs/gym_memberships.md

```diff
@@ -2,12 +2,108 @@
 
 HTTP Verb | Endpoint                   | Description                | Link
 ----------|----------------------------|----------------------------|---------------------------
+GET       | `/users/{user_id}/gym_members?yelp_gym_id={yelp_gym_id}` | Get users that belong to the same gym.     | [Link](#get-users-at-same-gym)
```

Comment:
I'd say we leave the query parameters out of this endpoint

Reply:
Updated accordingly